comfy_cli/file_utils.py

```diff
@@ -86,61 +88,139 @@ def download_file(url: str, local_filepath: pathlib.Path, headers: Optional[dict
             raise DownloadException(f"Failed to download file.\n{status_reason}")
 
 
+def _load_comfyignore_spec(ignore_filename: str = ".comfyignore") -> Optional[PathSpec]:
+    if not os.path.exists(ignore_filename):
+        return None
+    try:
+        with open(ignore_filename, encoding="utf-8") as ignore_file:
+            patterns = [line.strip() for line in ignore_file if line.strip() and not line.lstrip().startswith("#")]
+    except OSError:
+        return None
+
+    if not patterns:
+        return None
+
+    return PathSpec.from_lines("gitwildmatch", patterns)
+
+
+def list_git_tracked_files(base_path: Union[str, os.PathLike] = ".") -> list[str]:
+    try:
+        result = subprocess.check_output(
+            ["git", "-C", os.fspath(base_path), "ls-files"],
+            text=True,
+        )
+    except (subprocess.SubprocessError, FileNotFoundError):
+        return []
+
+    return [line for line in result.splitlines() if line.strip()]
+
+
+def _normalize_path(path: str) -> str:
+    rel_path = os.path.relpath(path, start=".")
+    if rel_path == ".":
+        return ""
+    return rel_path.replace("\\", "/")
+
+
+def _is_force_included(rel_path: str, include_prefixes: list[str]) -> bool:
```

Comment:
when `zip_files `  walks files, it checks `_is_force_included()` before respecting .comfyignore. So anything listed in `tool.comfy.includes` (which is where we surface required assets like `__init__.py` or `web/*` is forced back into the archive even if it matches an ignore rule. The new `test_zip_files_force_include_overrides_ignore` exercises that exact flow.